.github/workflows/release.yml

```diff
@@ -9,6 +9,9 @@ jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
+      - name: Set up QEMU
```

Comment:
This is needed for `buildx` and `goreleaser` to build multi-arch. Also documented in  https://goreleaser.com/cookbooks/multi-platform-docker-images/